src/components/MultiHopTrade/helpers.ts

```diff
@@ -29,42 +28,18 @@ export const getMixpanelEventData = () => {
   if (!buyAsset?.precision) return
 
   const assets = selectAssets(state)
-  const _donationAmountUserCurrency = selectQuoteDonationAmountUserCurrency(state)
-  const _donationAmountUsd = selectQuoteFeeAmountUsd(state)
+  const shapeShiftFeeUserCurrency = selectQuoteAffiliateFeeUserCurrency(state)
+  const shapeshiftFeeUsd = selectQuoteFeeAmountUsd(state)
   const sellAmountBeforeFeesUsd = selectSellAmountUsd(state)
   const sellAmountBeforeFeesUserCurrency = selectSellAmountUserCurrency(state)
   const buyAmountBeforeFeesCryptoPrecision = selectBuyAmountBeforeFeesCryptoPrecision(state)
   const sellAmountBeforeFeesCryptoPrecision = selectSellAmountBeforeFeesCryptoPrecision(state)
-  const willDonate = selectWillDonate(state)
   const swapperName = selectActiveSwapperName(state)
   const activeQuote = selectActiveQuote(state)
 
   const compositeBuyAsset = getMaybeCompositeAssetSymbol(buyAsset.assetId, assets)
   const compositeSellAsset = getMaybeCompositeAssetSymbol(sellAsset.assetId, assets)
 
-  const {
-    shapeShiftFeeUserCurrency,
-    shapeshiftFeeUsd,
-    donationAmountUserCurrency,
-    donationAmountUsd,
-  } = (() => {
-    if (swapperName === SwapperName.Thorchain) {
-      return {
-        shapeshiftFeeUsd: _donationAmountUsd,
-        shapeShiftFeeUserCurrency: _donationAmountUserCurrency,
-        donationAmountUsd: undefined,
-        donationAmountUserCurrency: undefined,
-      }
-    }
-
-    return {
-      shapeshiftFeeUsd: undefined,
-      shapeShiftFeeUserCurrency: undefined,
-      donationAmountUsd: _donationAmountUsd,
-      donationAmountUserCurrency: _donationAmountUserCurrency,
-    }
-  })()
-
```

Comment:
Again, weird and most likely off branching. Why do we use the `shapeshiftFee` for THOR, but the `donation` one, I have no idea. This stems from the old notion of a THORChain affiliate fees flag, and in https://github.com/shapeshift/web/pull/5403, monkey see, monkey did, but this really have no reason to exist.